addons/account/static/src/components/product_label_section_and_note_field/product_label_section_and_note_field.js

```diff
@@ -142,7 +142,14 @@ export class ProductLabelSectionAndNoteField extends Many2OneField {
     }
 
     get label() {
-        return this.props.record.data.name;
+        var label = this.props.record.data.name;
```

Comment:
```suggestion
        let label = this.props.record.data.name;
```
Generally, don't use `var` :wink: